.circleci/config.yml

```diff
@@ -116,6 +116,18 @@ references:
       command: |
         echo "Loading wiremock mappings..."
         find spec/wiremock/prison-api/mappings/*.json -exec curl --request POST --url http://localhost:8888/__admin/mappings/ --header 'content-type: application/json' --data-binary "@{}" \;
+  _notify_notify_sentry_release: &notify_notify_sentry_release
```

Comment:
Do we need notify twice in the step definition?

Reply:
```suggestion
  _notify_sentry_release: &notify_sentry_release
```

Reply:
Good spot, bad `s//`